### PR TITLE
Explaining the two major ways to register an event

### PR DESCRIPTION
**Pull request by ThomasLandauer**
Just some by-product of https://github.com/doctrine/orm/pull/9039 :-) - see https://github.com/doctrine/orm/pull/9039#issuecomment-933359802

### CODE REVIEW

**Comment by derrabus** [comment]
Please target 2.10.x. 2.9 is closed.

File: docs/en/reference/events.rst [hunk 1]
```diff
@@ -121,6 +121,16 @@ Now you can test the ``$eventSubscriber`` instance to see if the
         echo 'pre foo invoked!';
     }
 
+Registering Events
+~~~~~~~~~~~~~~~~~~
+
+There are two ways to register an event:
+
+* *All events* can be registered by calling ``$eventManager->addEventListener()``
+or ``eventManager->addEventSubscriber()`` (see [below](#listening-and-subscribing-to-lifecycle-events))
```
Comment (greg0ire) [inline docs/en/reference/events.rst:130]: This syntax looks like markdown… does it really work in RST?

**Comment by ThomasLandauer** [comment]
See https://github.com/doctrine/orm/pull/9102

File: docs/en/reference/events.rst (hunk 1 above)
Comment (ThomasLandauer) [reply docs/en/reference/events.rst:130]: Thanks! Please see https://github.com/doctrine/orm/pull/9102

**Pull request closed without merging**